cloud_governance/common/mails/mail_message.py

```diff
@@ -239,7 +243,6 @@ def get_long_run_alert(self, days: int, user: str, jira_id: str):
                 <div>
                     <p>This is a message to alert you that in {days} days, the cloud request is expiring.</p>
```

Comment:
Hi {user},

You project budget request {ticket_id} will be expired in {days} days.